config/versions/versions.go

```diff
@@ -220,6 +219,14 @@ func (m *manager) registerVersion(ver int, v any) {
 	m.versions[ver] = v
 }
 
+// Version returns a version registered by init or nil if nothing has been registered with that version number
+func (m *manager) Version(version int) any {
```

Comment:
Suggestion/look away: int -> uint64

Reply:
This would cause overflow issues to mitigate... but fortunately, it actually *should* be `uint16`, which would not.

Fixed https://github.com/gbjk/gocryptotrader/commit/e2e552016d